scripts/leaf-scripts/showdbvers.php

```diff
@@ -0,0 +1,29 @@
+<?php

```

Comment:
What's this file for?

Reply:
Added usage text to this and the dbvers file that I had added